src/components/TodaysFood.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+function TodaysFood(props){
+    let todaysFood = props.foodList.filter((foods)=>{
+        return foods.quantity > 0
```

Comment:
this is actually a very clever way to go around the todays food iteration. Nice!